tgui/packages/tgui/interfaces/CentcomPodLauncher.js

```diff
@@ -1,518 +1,1125 @@
-              onClick={() => act('styleMissile')} />
-            <Button
-              content="Syndicate Missile"
-              selected={data.styleChoice === 8}
-              tooltip={multiline`
-                A large blood-red missile. Combos well with missile mode,
-                so the missile doesnt stick around after landing.
-              `}
-              onClick={() => act('styleSMissile')} />
-            <Button
-              content="Supply Crate"
-              selected={data.styleChoice === 9}
-              tooltip="A large, dark-green military supply crate."
-              onClick={() => act('styleBox')} />
-            <Button
-              content="HONK"
-              selected={data.styleChoice === 10}
-              tooltip="A colorful, clown inspired look."
-              onClick={() => act('styleHONK')} />
-            <Button
-              content="~Fruit"
-              selected={data.styleChoice === 11}
-              tooltip="For when an orange is angry"
-              onClick={() => act('styleFruit')} />
-            <Button
-              content="Invisible"
-              selected={data.styleChoice === 12}
-              tooltip={multiline`
-                Makes the supplypod invisible! Useful for when you want to
-                use this feature with a gateway or something. Combos well
-                with the "Stealth" and "Quiet Landing" effects.
-              `}
-              onClick={() => act('styleInvisible')} />
-            <Button
-              content="Gondola"
-              selected={data.styleChoice === 13}
-              tooltip={multiline`
-                This gondola can control when he wants to deliver his supplies
-                if he has a smart enough mind, so offer up his body to ghosts
-                for maximum enjoyment. (Make sure to turn off bluespace and
-                set an arbitrarily high open-time if you do!
-              `}
-              onClick={() => act('styleGondola')} />
+                selected={
+                  option.key
+                    ? data.reverseOptionList[option.key]
+                    : data.reverseOptionList[option.title]
+                }
+                tooltip={option.title}
+                tooltipOverrideLong
+                onClick={() => act('reverseOption', {
+                  reverseOption: option.key
+                    ? option.key
+                    : option.title })} />
+            ))}
+          </Flex.Item>
+        </Flex>
+      )}
+    </Section>
+  );
+};
+
+const saveDataToPreset = (id, data) => {
+  storage.set("podlauncher_preset_"+id, data);
+};
+
+const loadDataFromPreset = (id, context) => {
+  const { act, data } = useBackend(context);
+  act('loadDataFromPreset', { payload: storage.get("podlauncher_preset_"+id) });
+};
+
+const newPreset = (presetName, hue, data) => {
+  const current = getPresets();
+  let id = CreateUuid();
+  current.push({ id: id, title: presetName, hue: hue });
```

Comment:
Properties with the same name as a variable can be combined.
```suggestion
  current.push({ id, title: presetName, hue });
```